# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fbset/install.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2003-03-22 11:05:26.000000000 +0100
+++ ./Makefile	2003-03-22 11:06:04.000000000 +0100
@@ -25,6 +25,7 @@
 
 install:	fbset
 		if [ -f /sbin/fbset ]; then rm /sbin/fbset; fi
+		if [ -f /usr/sbin/fbset ]; then rm /usr/sbin/fbset; fi
-		$(INSTALL) fbset /usr/sbin
+		$(INSTALL) fbset /bin
 		$(INSTALL) fbset.8 /usr/man/man8
 		$(INSTALL) fb.modes.5 /usr/man/man5
